packages/payment-detection/package.json

```diff
@@ -47,20 +47,21 @@
     "@requestnetwork/utils": "0.38.0",
     "axios": "0.27.2",
     "ethers": "5.5.1",
-    "graphql": "15.5.1",
-    "graphql-request": "3.4.0",
-    "graphql-tag": "2.12.4",
+    "graphql": "16.8.1",
+    "graphql-request": "6.1.0",
+    "graphql-tag": "2.12.6",
     "lodash": "4.17.21",
     "satoshi-bitcoin": "1.0.4",
     "tslib": "2.5.0"
   },
   "devDependencies": {
-    "@graphql-codegen/cli": "1.21.5",
-    "@graphql-codegen/typescript": "1.22.1",
-    "@graphql-codegen/typescript-document-nodes": "1.17.12",
-    "@graphql-codegen/typescript-graphql-request": "3.2.1",
-    "@graphql-codegen/typescript-operations": "1.18.0",
-    "@graphql-codegen/typescript-resolvers": "1.19.2",
+    "@babel/helper-get-function-arity": "^7.16.7",
```

Comment:
Without this new package generating the graphql code result in the following error:
```
Error: Cannot find module '@babel/helper-get-function-arity'
```

I cannot find anything online regarding this, which surprises me, but adding this package solves the issue.